### DISCUSSION TITLE
Disable routing and export pure static HTML

### DISCUSSION THREAD

**Discussion by wishnuprathikantam** [react-static/react-static#1662] [Q&A]
Is it possible to export the build without any routing capabilities as a pure HTML file with all the required CSS, JS included.
Right now, I think it loads vendor files based on the route can I change this behavior. The end goal here is to render the page correctly with all the styles applied on the browser without the need for a dev-server or hosting. 

Any help is appreciated. Thanks in advance.

**Comment by SleeplessByte** [comment]
You mean a single HTML file that has all the routes included?

That's not possible.

**Reply by wishnuprathikantam** [reply]
Separate HTML File(s) for each page with no javascript routing. I should be able to link the files using a tag href=/page.html. Also, js shouldn't contain any rendered HTML inside it. One way I think of is to rewrite the site without using react (using Vanilla JS or JQuery) just curious to know if this is possible with react and react-static.

**Reply by SleeplessByte** [reply]
The standard output of a production build is hostable without _any_ server. It does exactly what you say: generate an HTML page per route.

**Reply by wishnuprathikantam** [reply]
Firstly thank you for the help.
This is what I'm exactly trying to do. Convert the generated output to a WordPress theme. I don't know if this works so I've written a basic layout using the mantine.dev components and trying to convert it to an HTML output using react-static. I'll use the generated output to make a WordPress theme.
I've generated a file and was trying to convert it into a WordPress theme. I've copied the generated HTML and when I've linked the generated scripts & stylesheets but the js is actually looking for routeInfo.json. I've managed to skip this error by removing routing and generating a single page with some test data hardcoded (Think of this page as my basic template, FYI not using any dynamic routes or data) without any routing. While trying to convert the generated output to a theme I've removed all the hardcoded data from the index.html file and converted it to Wordpress theme but the final theme is able to render the data I've removed from the HTML file. I've noticed the generated js file contains the strings and HTML data as well and is been rendered in the final output.
